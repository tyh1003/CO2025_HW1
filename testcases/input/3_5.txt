15
78 54 77 52 92 28 62 37 56 93 2 25 29 63 69
