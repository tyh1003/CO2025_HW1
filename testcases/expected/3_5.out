2 25 28 29 37 52 54 56 62 63 69 77 78 92 93 
